diff --git a/lib/index.d.ts b/lib/index.d.ts
index 6ed1a1ed725cee77b83840c20024af4ec6a12b40..29437719aeba0df39df730c88e69accf1dcf0455 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -10,7 +10,7 @@ declare const plugin: {
                         namespace: string;
                     };
                     rules: {
-                        '@ms-cloudpack/no-deprecated': TSESLint.RuleModule<"forbidden", [({
+                        'no-deprecated': TSESLint.RuleModule<"forbidden", [({
                             ignoredSelectors?: string[];
                         } | undefined)?], unknown, TSESLint.RuleListener>;
                     };
@@ -26,7 +26,7 @@ declare const plugin: {
                         namespace: string;
                     };
                     rules: {
-                        '@ms-cloudpack/no-deprecated': TSESLint.RuleModule<"forbidden", [({
+                        'no-deprecated': TSESLint.RuleModule<"forbidden", [({
                             ignoredSelectors?: string[];
                         } | undefined)?], unknown, TSESLint.RuleListener>;
                     };
@@ -43,7 +43,7 @@ declare const plugin: {
         namespace: string;
     };
     rules: {
-        '@ms-cloudpack/no-deprecated': TSESLint.RuleModule<"forbidden", [({
+        'no-deprecated': TSESLint.RuleModule<"forbidden", [({
             ignoredSelectors?: string[];
         } | undefined)?], unknown, TSESLint.RuleListener>;
     };
diff --git a/lib/index.js b/lib/index.js
index b1a696cc9609f3945e457ec24a8fc4262a4f0c35..8a72ec6f8c39aa7ba71280dd50b1688913ff83d8 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -5,7 +5,7 @@ const { name, version } = JSON.parse(fs.readFileSync(path.resolve(import.meta.di
 const basePlugin = {
     meta: { name, version, namespace: '@ms-cloudpack' },
     rules: {
-        '@ms-cloudpack/no-deprecated': noDeprecated,
+        'no-deprecated': noDeprecated,
     },
     // Use `satisfies` to preserve the original property names
 };
