diff --git a/node_modules/lage/dist/runners/NpmScriptRunner.js b/node_modules/lage/dist/runners/NpmScriptRunner.js
index 143ea2d..bcf2075 100644
--- a/node_modules/lage/dist/runners/NpmScriptRunner.js
+++ b/node_modules/lage/dist/runners/NpmScriptRunner.js
@@ -79,16 +79,17 @@ class NpmScriptRunner {
         await new Promise((resolve, reject)=>{
             childProcess = (0, _childProcess.spawn)(npmCmd, npmRunArgs, {
                 cwd: target.cwd,
+                ...(process.platform === 'win32' && { shell: true }),
                 stdio: [
                     "inherit",
                     "pipe",
                     "pipe"
                 ],
                 env: {
-                    ...process.env,
                     ...process.stdout.isTTY && {
                         FORCE_COLOR: "1"
                     },
+                    ...process.env,
                     ...npmRunNodeOptions && {
                         NODE_OPTIONS: npmRunNodeOptions
                     },
